ndarray/src/test/java/org/tensorflow/ndarray/impl/sparse/DoubleSparseNdArrayTest.java

```diff
@@ -302,4 +303,15 @@ public void testSlice() {
                     .forEachIndexed(
                         (lidx, f) -> assertEquals(expected[i.getAndIncrement()], f.getDouble())));
   }
+
+  @Test
+  public void testToString() {
+    DoubleNdArray ndArray = StdArrays.ndCopyOf(dense2DArray);
+    DoubleSparseNdArray instance =
+            DoubleSparseNdArray.create(DimensionalSpace.create(ndArray.shape()));
+    instance.fromDense(ndArray);
+    Assertions.assertEquals("DoubleSparseNdArray(defaultValue=0.0,numElements=2,shape=[3, 4])",instance.toString());
```

Comment:
It might be a personal preference but I think elements split by commas are more readable if they are preceded by a space: `DoubleSparseNdArray(defaultValue=0.0, numElements=2, shape=[3, 4])`

Reply:
Sure, I can do that.

Reply:
Done.